spikeinterface_gui/tests/iframe/iframe_test.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
```

Comment:
same for this.
We should have a sub folder for this no ?

Reply:
it is in a subfolder